### PR TITLE
docs(1): thoughtless oss always dies

### LINKED ISSUE #1: The thoughtless OSS always dies
### Content Type

Article

### Article Description

The recent changes in licensing of Cockroach db have raised concerns and highlighted the importance of a well-considered open source software model.

### Target Audience

startup CEO

### References/Resources

https://news.ycombinator.com/item?id=41256222
https://techcrunch.com/2024/08/15/cockroach-labs-shakes-up-its-licensing-to-force-bigger-companies-to-pay/
https://x.com/PeterZaitsev/status/1824088154795802642
https://www.cockroachlabs.com/blog/enterprise-license-announcement/

### Examples

_No response_

### Special Instructions

_No response_

### PR DESCRIPTION
**Pull request by hunxjunedo**
fixes #1
/claim #1 

this article is written with high effort:
- importance is given to research-based material 
- contains case study to gradually build a conclusion for the reader
- most of the part is written by self, with AI used at few parts to check facts, and then manually rephrased to sound human.
- has links of relevant articles and blogs to support the points
- has comparisons to stress on the facts
- has been written with careful consideration of the target audience, ensuring it's engaging for them

**therefore I'd like to get assigned, I'm working on adding media and illustration**

# Pull Request Template

## Writer's Checklist

- [x] **Follow Writing Structure**
  - [x] I have used more than two H2 subtitles to organize content.
  - [x] Include an introduction with at least two paragraphs before the first H2 section.
  - [x] Use bold, italic, underline, notes, quotes, TLDRs, and key points as needed.
  - [ ] Include `code elements` where appropriate.
  - [x] Write the article in Markdown format.

- [x] **Fact-Check**
  - [x] I have verified all facts and data points included in the article.

- [ ] **Assets**
  - [ ] Save images in the `/assets` folder.
  - [ ] Follow naming conventions: `YYYYMMDD_title_of_the_article_IMG_NAME_NO.png`.

- [ ] **Interlinking**
  - [ ] Use `CTRL+F` to search for relevant keywords on:
    - [Daytona Dotfiles Sitemap](https://www.daytona.io/sitemap-dotfiles.xml)
    - [Daytona Definitions Sitemap](https://www.daytona.io/sitemap-definitions.xml)
  - [x] Interlink keywords using the appropriate URLs.

- [ ] **Glossary/Definitions**
  - [ ] Identify and mark new terms that could be added to the Definitions page.

- [x] **Review and Edit**
  - [x] Read the article out loud to catch any awkward phrasing.
  - [x] Run the draft through [Grammarly](https://grammarly.com) or a similar tool to correct grammar and spelling errors.
  - [x] Double-check adherence to the style guide and repository guidelines.

### CODE REVIEW

File: articles/20240821_oss_model.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Surviving as Open Source These Days: Harder Than You Think"
+description: "Read along as we discuss common failiures in OSS models, and how to plan one which is perfect for your startup."
```
Comment (nkkko) [inline articles/20240821_oss_model.md:3]: Improve this please

File: articles/20240821_oss_model.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- What does this all mean for the future of open-source software?
+- What should tech-startups learn from it?
+
+This article explores the licensing changes in prominent OSS projects, focusing on CockroachDB’s recent shift. We’ll discuss the motivations behind these decisions, identify common patterns, and provide insights into what constitutes a robust and sustainable OSS model.
+
+## Case Study
+
+### Cockroach DB
+
+The very first open-source release of CockroachDB was done under the Apache 2.0 License, after which it gained extreme adoption and huge people/community support. However, CockroachDB faced abuse by cloud-providers, so in 2019, Cockroach Labs changed its licensing to the Business Source License, defining the very notion of open access and finding a way to combine it with control in one commercially available instrument. Recently, they introduced a proprietary model in the CockroachDB software license. Although strategic, this move brought controversy to the community since it deviated from the primary values of the project—being open-sourced.
```
Comment (nkkko) [inline articles/20240821_oss_model.md:24]: do the interlinking from our sitemaps, at least 3 pages, consider adding a new definition realted to the topic: oss licence or similar

**Review by nkkko — CHANGES_REQUESTED** [review]
Well written, please read once more and improve where needed. Implement the full checklist. Add at least one definition, author, etc. Please follow the instructions in the readme.

**Comment by hunxjunedo** [comment]
@nkkko one more thing, is there a need to add illustrations ? If so, then I need on design briefing (font, colors etc. if it's specific)

**Comment by hunxjunedo** [comment]
@nkkko I've applied the changes you suggested, let me know if there's anything else to change.

File: articles/20240821_oss_model.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+- **Promote Community Engagement**: Even with more restrictive licenses, it’s crucial to maintain a vibrant and engaged community. Open-source projects thrive on community contributions, which can drive innovation and improve the software. Start with a creative and skilled team of maintainers and contributors who know what is quality code. [Read:  Building a Community Before Launching Your Product](https://www.daytona.io/dotfiles/building-a-community-before-launching-your-product)
+
+- **Offer Clarity and Fairness**: The terms of the license should be clear, fair, and easy to understand. Ambiguity can lead to mistrust or misuse, which can harm the project in the long run. It would be a nightmare to find people taking advantages of some loopholes no one at your team even noticed. [Read: Open Source Licenses to Avoid](https://brainhub.eu/library/open-source-licenses-to-avoid)
+
+[Read More: Building a Successful Open Source Project](https://www.daytona.io/dotfiles/building-a-successful-open-source-project)
```
Comment (nkkko) [inline articles/20240821_oss_model.md:58]: please fix all of the read more links, all links need to be inline with text, so link appropriate keywords in the text.

**Review by nkkko — CHANGES_REQUESTED** [review]
Proofreading: There are some minor grammatical and spelling errors throughout the text that should be corrected (e.g., "failiures" in the description should be "failures").

Structure: The article could benefit from a more clearly defined structure with subheadings for each main point to improve readability.

Depth: Some sections, particularly the case studies, could be expanded to provide more detailed information and analysis.

Solutions: The "A good OSS Model" section could be expanded to provide more concrete strategies and examples of successful implementations.

Consistency: Ensure consistent formatting throughout the article, especially in the use of bold and italics.

Engagement: Consider adding more engaging elements like pull quotes, infographics, or bullet points to break up the text and highlight key information.

Conclusion: The "Final Thoughts" section could be strengthened to provide a more impactful conclusion that ties together the main points of the article.

Audience consideration: Clarify the target audience and adjust the tone and content accordingly.

**Comment by hunxjunedo** [comment]
@nkkko applied the requested changes. Please review.

**Comment by mojafa** [comment]
@hunxjunedo I like how you've incorporated the requests, like pull quotes, and bullet points. Please implement the full checklist. Add images where possible.

@nkkko Reckon there's need for images or inforgraphics on this? This article is well articulated and has data like MySQL vs MongoDB.

**Comment by nkkko** [comment]
Thanks @hunxjunedo for your work!

You can showcase your contribution on your GitHub profile: https://github.com/daytonaio/content/blob/main/CONTRIBUTING.md#showcase-your-contributions

**Pull request merged**